pennylane/math/multi_dispatch.py

```diff
@@ -870,6 +870,50 @@ def norm(tensor, like=None, **kwargs):
     return norm(tensor, **kwargs)
 
 
+@multi_dispatch(argnum=[0])
+def svd(tensor, like=None, **kwargs):
+    """Compute the singular value decomposition of a tensor in each interface.
+
+    For a matrix :math:`A`, the singular value decomposition consist of three matrices :math:`S`, :math:`U` and :math:`Vh`, such that:
+    .. math::
+
+        A = U . Diag(S) . Vh
+
+    Returns:
+        :math:`S`, :math:`U` and :math:`Vh`: full decomposition (if compute_uv is True or None)
+        :math:`S`: only the singular values (if compute_uv is False)
+
+    """
+    if like == "tensorflow":
+        from tensorflow.linalg import svd, adjoint
+
+        # Tensorflow results need some post-processing to keep it similar to other frameworks.
+
+        if kwargs.get("compute_uv", True):
+            S, U, V = svd(tensor, **kwargs)
+            return U, S, adjoint(V)
+        return svd(tensor, **kwargs)
+
+    if like == "jax":
+        from jax.numpy.linalg import svd
+
+    elif like == "torch":
+        # Torch is deprecating torch.svd() in favor of torch.linalg.svd().

```

Comment:
```suggestion
        # Torch is deprecating torch.svd() in favour of torch.linalg.svd().
```
Very minor, but just to follow out [Canadian spelling guideline](https://xanaduai-pennylane--5154.com.readthedocs.build/en/5154/development/guide/documentation.html#punctuation-and-spelling).